cpp/examples/ode_secir.cpp

```diff
@@ -76,12 +76,19 @@ int main()
 
     model.apply_constraints();
 
+    // Using default Integrator
+    mio::TimeSeries<double> secir = simulate(t0, tmax, dt, model);
+
+    /*
+    Example of using a different integrator
+    
     auto integrator = std::make_shared<mio::RKIntegratorCore>();
     integrator->set_dt_min(0.3);
     integrator->set_dt_max(1.0);
     integrator->set_rel_tolerance(1e-4);
     integrator->set_abs_tolerance(1e-1);
     mio::TimeSeries<double> secir = simulate(t0, tmax, dt, model, integrator);
+    */
 
     bool print_to_terminal = true;

```

Comment:
we should use the new print_table function here. However, this is not the main part of this issue and maybe we shift this to an own issue.